_playwright-tests/UI/TemplateCRUD.spec.ts

```diff
@@ -11,7 +11,7 @@ const repoNamePrefix = 'custom_repo-template';
 
 const repoName = `${repoNamePrefix}-aarch64-${randomName()}`;
 const repoNameX86 = `${repoNamePrefix}-x86_64-${randomName()}`;
-const repoNameIntrospect = `${repoNamePrefix}-introspect-only-${randomName()}`;
+const repoNameNoSnaps = `${repoNamePrefix}-introspect-only-${randomName()}`;
```

Comment:
Could you please use commit messages? E.g. this commit only has title "Change introspect only repo name " with no reasoning about why you did this change. It would make it much easier to review if you could explain why some changes are being done.

Reply:
yeah, would be good to a reason for this (in the commit or in the PR) as I don't really get why this commit was made 💭